infinite-slider.js

```diff
@@ -208,6 +208,18 @@
                 interactionCurrent.y += dy;
               }
             }
+            // had to add this to avoid getting undefined at line 211 when clicking google maps
+            // this seems like a hack, if you know of a better solution please delete this.
+            // *** start of added code
+               if (!interactionStart && !interactionCurrent) {
+                  interactionStart = interactionCurrent = { x: 0};
```

Comment:
Yeah I need that as well thank you! Also, is there anyway to prevent the infinite part -> make it end on the last slide, not jump to the beginning?